packages/flutter/lib/src/cupertino/text_selection.dart

```diff
@@ -41,17 +41,34 @@ const TextStyle _kToolbarButtonFontStyle = TextStyle(
   color: CupertinoColors.white,
 );
 
+/// The direction of the triangle next to the toolbar.
```

Comment:
``` The direction of the triangle attached to the toolbar

Defaults to showing the triangle downwards if sufficient space is available to show the toolbar above the text field. Otherwise, the toolbar will appear below the text field and the triangle's direction will be [up].
```

Reply:
Umm the comment above somehow broke github :)

I meant:
```dart
The direction of the triangle attached to the toolbar

Defaults to showing the triangle downwards if sufficient space is available to show the toolbar above the text field. Otherwise, the toolbar will appear below the text field and the triangle's direction will be [up].
```